File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        vector<int>ans1;
        vector<int>ans2;
        int len=max(s.length(),t.length());
        for(int i=0;i<len;i++){
            ans1.push_back(s[i]);
            ans2.push_back(t[i]);
        }
        // if(ans1.size()!=ans2.size()){
        //     return false;
        // }
        sort(ans1.begin(),ans1.end());
        sort(ans2.begin(),ans2.end());
        for(int i=0;i<ans2.size();i++){
            if(ans1[i]!=ans2[i]){
                return false;
            }
        }
        return true;
    }
};

File: 0242-valid-anagram/0242-valid-anagram.rs
impl Solution {
    pub fn is_anagram(s: String, t: String) -> bool {
        let mut chars: Vec<char> = s.chars().collect();
        chars.sort();
        let mut chars_2 : Vec<char> = t.chars().collect();
        chars_2.sort();
        if chars.len() != chars_2.len(){
            return false;
        } 
        for i in 0..chars.len() {
            if chars[i] != chars_2[i] {
               return false; 
            } 
        }
        return true;
    }
}